### PR TITLE
#161595854 User can add a travel document

### PR DESCRIPTION
**Pull request by ronaldndirangu**
#### What does this PR do?
This PR implements the add/upload document feature.

#### Description of Task to be completed?
- A user can upload the document to the cloud.
- A user can add the document to the database

#### How should this be manually tested?
- A Travel Admin/Super Admin for a location
 - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
      - Change into the project directory
      - Run the server `make start`
      - Login and navigate the documents page
      - Add a new document using the add file button
      - A modal will appear where you click the upload file button to choose the file.
      - In the modal type in the name of the document in the input field
      - Click the add button in the modal to upload/add the document
      - Upload image file or PDF.
      - A toast message should be displayed to indicate success or failure in case of any errors. 

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) and switch to the ft-add-travel-document-161595854 branch to test this functionality.
- Then run ```yarn db:rollmigrate```
- If using yarn start use the following .env variables
```
REACT_APP_CLOUNDINARY_API=https://api.cloudinary.com/v1_1/authors-haven/image/upload
REACT_APP_PRESET_NAME=iwbpjk8d
```

#### What are the relevant pivotal tracker stories?
[#161595854](https://www.pivotaltracker.com/story/show/161595854)

#### Screenshots (if appropriate)
screen shot 2018-11-07 at 15 41 14
screen shot 2018-11-07 at 15 41 29

#### Questions:

### CODE REVIEW

File: src/views/Documents/index.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Modal from '../../components/modal/Modal';
+import { openModal, closeModal } from '../../redux/actionCreator/modalActions';
+import DocumentsHeader from '../../components/DocumentsHeader';
+import NewDocumentForm from '../../components/Forms/NewDocumentForm';
+import {createDocument} from '../../redux/actionCreator/documentsActions';
```
Comment (loicemeyo) [inline src/views/Documents/index.jsx]: @ronaldndirangu properly space this import for consistency

Comment (ronaldndirangu) [reply src/views/Documents/index.jsx]: I have made the changes thanks for noting that.

**Comment by janetnim** [comment]
screen shot 2018-11-08 at 17 34 44
What happens if I want the file to be saved with the default name as shown above, instead of changing it?
Also, I cannot be able to view the document I've uploaded. Is this within the scope of your task?

**Comment by ronaldndirangu** [comment]
@janetnim You can just live it as is you just click the input box and don't change anything. To view the document is not part of this scope.

**Comment by Spectrumsun** [comment]
fix conflict @ronaldndirangu

**Comment by ronaldndirangu** [comment]
@Spectrumsun I have resolved the conflicts.

**Pull request merged**